src/Details/Details.css

```diff
@@ -0,0 +1,5 @@
+.Gallery {
```

Comment:
Mmm...why the `Details` component starts with a `Gallery` class? it should be `Details` instead for consistency reasons!